host/tests/integration/components/schema-inspector-test.gts

```diff
@@ -96,4 +139,186 @@ module('Integration | schema-inspector', function (hooks) {
     assert.dom('[data-test-field="firstName"] input').doesNotExist();
     assert.dom('[data-test-field="lastName"] input').doesNotExist();
   });
-});
\ No newline at end of file
+
+  test('Can render field that uses base field card', async function (assert) {
+    await renderComponent(
+      class TestDriver extends GlimmerComponent {
+        <template>
+          <SchemaInspector
+            @module={{singleCardModule}}
+            @src={{singleCardSrc}}
+            @inspector={{inspector}}
+            @path="/"
+          />
+        </template>
+      }
+    );
+
+    assert.dom('[data-test-field="firstName"]').containsText('firstName: contains runtime-spike/lib/string card');
+    assert.dom('[data-test-field="firstName"] a').doesNotExist();
+    assert.dom('[data-test-field="lastName"]').containsText('lastName: contains runtime-spike/lib/string card');
+    assert.dom('[data-test-field="lastName"] a').doesNotExist();
+  });
+
+  test('Can render field that uses a user defined external card', async function (assert) {
+    await renderComponent(
+      class TestDriver extends GlimmerComponent {
+        <template>
+          <SchemaInspector
+            @module={{usesExternalCardModule}}
+            @src={{usesExternalCardSrc}}
+            @inspector={{inspector}}
+            @path="/"
+          />
+        </template>
+      }
+    );
+    assert.dom('[data-test-field="author"]').containsText("author: contains 'Person' card of ./person");
+    assert.dom('[data-test-field="author"] a[href="/?path=person"').containsText("'Person' card of ./person");
+  });
+
+  test('Can render field that uses card internal to the module', async function(assert) {
+    await renderComponent(
+      class TestDriver extends GlimmerComponent {
+        <template>
+          <SchemaInspector
+            @module={{multipleCardsModule}}
+            @src={{multipleCardsSrc}}
+            @inspector={{inspector}}
+            @path="/"
+          />
+        </template>
+      }
+    )
+
+    await click('.card-button[data-test-card-name="Post"]');
+    assert.dom('[data-test-field="author"]').containsText("author: contains 'Person' card");
+  });
+
+  skip('Can render field that uses contains-many field');
+  skip('Can render field that uses computed field');
+});
+
+
+// UGH I don't have a good answer for this--we want both
+// the card source and the evaluated card modules (which
+// requires a babel build step). So we need to keep these
+// sources and their corresponding modules in sync
```

Comment:
the testing for the schema inspector is not pleasant. the schema inspector now wants to consume both the source files and their related modules (this is the work that babel is doing in the worker). I wasn't sure how to construct these without having to repeat myself for both the source and the modules in our tests.